src/ui/main.cpp

```diff
@@ -82,21 +92,25 @@ static void setupCommandLineParser(QCommandLineParser* parser) {
     parser->addHelpOption();
     parser->addPositionalArgument("sound_file", QApplication::translate("main", "File to load."));
 
-    parser->addOption(
-        {"export",
-         QApplication::translate("main", "Create a wav file from the given SFXR file and exit.")});
+    parser->addOption({"export",
+                       QApplication::translate(
+                           "main", "Creates a wav file from the given SFXR file and exits.")});
     parser->addOption(
         {{"o", "output"},
-         QApplication::translate("main", "Set the file to export to if --export is given."),
-         "file"});
+         QApplication::translate("main", "Specifies the path for the file created with --export."),
```

Comment:
For the sake of conciseness, I think we can remove the "Specifies the" part of the help texts. Something like this:
```suggestion
         QApplication::translate("main", "Path for the file created with --export."),
```

Reply:
I'd rather not have the help texts be predicateless since they're describing the option, not just it's parameter. (I'd also rather have them in the imperative mood but that would be inconsistent with Qt:s help text for --help.) "Sets" would be a bit shorter but doesn't sound as good to me.

Reply:
OK, makes sense, merging it.